spec/support/shared/spree_builders.rb

```diff
@@ -1,5 +1,4 @@
 RSpec.shared_context 'spree_builders' do
-  # TODO: This starts already to be out of hand. Think about fixing this
```

Comment:
I decided to get rid of it and create an issue that would represent the potential need of refactor. What do you think?